syntax/elixir.vim

```diff
@@ -22,13 +22,9 @@ syn match elixirKeyword '\(\.\)\@<!\<\(for\|case\|when\|with\|cond\|if\|unless\|
 syn match elixirKeyword '\(\.\)\@<!\<\(exit\|raise\|throw\|after\|rescue\|catch\|else\)\>'
 syn match elixirKeyword '\(\.\)\@<!\<\(quote\|unquote\|super\|spawn\|spawn_link\|spawn_monitor\)\>'
 
-" Functions used on guards
-syn keyword elixirKeyword contained is_atom is_binary is_bitstring is_boolean
-syn keyword elixirKeyword contained is_float is_function is_integer is_list
-syn keyword elixirKeyword contained is_map is_nil is_number is_pid is_port
-syn keyword elixirKeyword contained is_record is_reference is_tuple is_exception
-syn keyword elixirKeyword contained abs bit_size byte_size div elem hd length
-syn keyword elixirKeyword contained map_size node rem round tl trunc tuple_size
+" Kernel functions
+syn match elixirKernelFunction contained containedin=elixirGuard '\<is_atom\|is_binary\|is_bitstring\|is_boolean\|is_float\|is_function\|is_integer\|is_list\|is_map\|is_nil\|is_number\|is_pid\|is_port\>\([ (]\)\@='
```

Comment:
In theory without the group the `\<` and `\>` are not applied to each word in the list. So, this: `\<\(A\|B\)\>` is equal to `\(\<A\>\|\<B\>\)`, whilst `\<A\|B\>` isn't.